build.sbt

```diff
@@ -49,23 +43,19 @@ val buildDebSettings = Seq(
   riffRaffArtifactResources += (file("cloudformation/membership-attribute-service.yaml") -> "cloudformation/membership-attribute-service.yaml"),
   Universal / javaOptions ++= Seq(
```

Comment:
updates copied from https://github.com/guardian/memsub-promotions/blob/2dd3e802a0fb580d84207c2fad0758d147b1608a/build.sbt#L43-L48
we also use in payment api and support admin console, but with even fewer parameters